vita3k/gui/src/about_dialog.cpp

```diff
@@ -131,7 +131,7 @@ void draw_about_dialog(GuiState &gui, EmuEnvState &emuenv) {
     const float STAFF_COLUMN_POS(HALF_WINDOW_WIDTH - (STAFF_LIST_SIZE.x / 2.f));
 
     ImGui::SetCursorPosX(STAFF_COLUMN_POS);
-    if (ImGui::BeginTable("##vita3k_staff_table", STAFF_COLUMN_COUNT, ImGuiTableFlags_ScrollY | ImGuiTableFlags_BordersV | ImGuiTableFlags_NoSavedSettings, STAFF_LIST_SIZE)) {
+    if (ImGui::BeginTable("##vita3k_staff_table", STAFF_COLUMN_COUNT, ImGuiTableFlags_ScrollY | ImGuiTableFlags_BordersV | ImGuiTableFlags_NoSavedSettings | ImGuiTableFlags_SizingFixedFit, STAFF_LIST_SIZE)) {
```

Comment:
Can you explain the reason for this?

Reply:
I remember the emulator crashing when opening the "about" dialog, but it does not happen anymore. I guess it's not necessary anymore.